department-of-veterans-affairs/caseflow
# Docs Redesign

Resolves #4599 

To test:
1. View the hearing worksheet and confirm the layout matches the AC
1. Confirm the layouts of the daily docket and the upcoming hearings page weren't inadvertently changed

<img width="1016" alt="screen shot 2018-04-26 at 12 05 45 pm" src="https://user-images.githubusercontent.com/4306128/39317678-2ef79242-494a-11e8-9350-ea1f3b362c36.png">
